docs/targets/zendesk.md

```diff
@@ -0,0 +1,68 @@
+# Event target for Zendesk
+
+This event target recieves [CloudEvents][ce] and interfaces with Zendesk by either creating a new Ticket or placing a new Tag on a Ticket depending on the CloudEvent type.
+
+## Prerequisite: API token
+
+A Zendesk API key is required to utilize this target. The steps to obtain a key are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
+
+## Deploying an instance of the Target
```

Comment:
Please ensure consistent capitalizations.  Each word should be capitalized _except_ for prepositions.